tests/contrib/test_appengine.py

```diff
@@ -557,6 +656,24 @@ def tearDown(self):
         self.testbed.deactivate()
         httplib2.Http = self.httplib2_orig
 
+    def test_in_error(self):
+        # Note: as far as I can tell this branch is never reached. _in_error
```

Comment:
I usually say `NOTE`, also `I` isn't very helpful to use in a comment like this.

Also it seems like this note should be an issue filed and the absence of the value being set should be addressed.